src/main/java/core/basesyntax/shape/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.shape;
+
+import core.basesyntax.type.Color;
+
+public class IsoscelesTrapezoid extends Shape {
+    private double base1;
+    private double base2;
+    private double height;
+
+    public IsoscelesTrapezoid(Color color, double base1, double base2, double height) {
+        super(color);
+        this.base1 = base1;
+        this.base2 = base2;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return (this.height / 2) * (this.base1 + this.base2);
```

Comment:
The formula for the area of an isosceles trapezoid should be (base1 + base2) * height / 2.